### PR TITLE
Add resources documentation

### PR DESCRIPTION
**Pull request by amir-hadzic**
Added documentation for the resources waterfall and resources table.

### CODE REVIEW

**Comment by amir-hadzic** [comment]
localhost_8000_experience_resources_

File: docs/experience/resources.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+title: Resources Timing
```
Comment (adnanrahic) [inline docs/experience/resources.md]: Resources -> Resource

Whenever you use the word **resource** with other words, it's not needed to have the plural of "resource". So, resource data and resource waterfall is correct, but also much easier to say compared to "resources data".

File: docs/experience/resources.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+description: How Sematext Experience collects external resources data such as downloaded images, CSS files, and Javascript files
+-----
+
+Experience collects resources data such as downloaded images, CSS and Javascript files. When someone visits your website the browser will begin downloading resources. These resources can have a big impact on your website loading speed.
```
Comment (adnanrahic) [inline docs/experience/resources.md]: resources data => resource data
Javascript => JavaScript
website loading speed => website's loading speed

File: docs/experience/resources.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+The same process happens in Single Page Apps when user navigates to a different part of the web app.
```
Comment (adnanrahic) [inline docs/experience/resources.md]: Single Page Apps when user navigates => single-page apps when a user navigates

File: docs/experience/resources.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+There are two ways to see resources performance data in Experience.
```
Comment (adnanrahic) [inline docs/experience/resources.md]: resources performance => resource performance

File: docs/experience/resources.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+
+## Resources Waterfall
```
Comment (adnanrahic) [inline docs/experience/resources.md]: Resources Waterfall => Resource Waterfall

File: docs/experience/resources.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+Open the details page for some page load to access the Resources Waterfall table.
```
Comment (adnanrahic) [inline docs/experience/resources.md]: Open the details page for some page load to access the Resources Waterfall table. => Open the page-load details page to access the Resource Waterfall table.

File: docs/experience/resources.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+
+This table shows downloaded resources ordered by the time when resource got discovered. Each row shows the different stages of the download process.
```
Comment (adnanrahic) [inline docs/experience/resources.md]: This table shows downloaded resources ordered by the time when resource got discovered. Each row shows the different stages of the download process. => This table shows downloaded resources ordered by the time they were discovered. Each row shows different stages of the download process.

File: docs/experience/resources.md [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+
+You can sort the waterfall by Size, Load Time, Time to Loaded, and Time Started (Waterfall).
+
+Let's explain what each of these mean:
+
+ * Size is the transferred size of the resource. Size will be "Unknown" if it's not available due to <a href="#cross-origin-resource-sharing-cors-restrictions">CORS restrictions</a>.
```
Comment (adnanrahic) [inline docs/experience/resources.md]: Size is => **Size** -

File: docs/experience/resources.md [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+ * Load Time is the time it took to download the resource once the fetch started
```
Comment (adnanrahic) [inline docs/experience/resources.md]: Load Time is => **Load Time** -

File: docs/experience/resources.md [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+ * Time to Loaded is the time from when the page load got initiated to when this resource got downloaded
```
Comment (adnanrahic) [inline docs/experience/resources.md]: Time to Loaded is => **Time to Loaded** -

File: docs/experience/resources.md [hunk 11]
```diff
@@ -0,0 +1,70 @@
...
+ * Time Started is the time when browser queued this resource for download
```
Comment (adnanrahic) [inline docs/experience/resources.md]: Time Started is => **Time Started** -

File: docs/experience/resources.md [hunk 12]
```diff
@@ -0,0 +1,70 @@
...
+
+<img
+  class="content-modal-image"
+  alt="Resources Waterfall Screenshot"
+  src="../../images/experience/waterfall.png"
+  title="Resources Waterfall"
+/>
+
+## Resources Overview
```
Comment (adnanrahic) [inline docs/experience/resources.md]: Resources Overview => Resource Overview

File: docs/experience/resources.md [hunk 13]
```diff
@@ -0,0 +1,70 @@
...
+
+The Top Resources table on the Overview page shows the site-wide performance for resources. The table is sortable by each column.
+
+ * Hits is the number of times the resource got requested
```
Comment (adnanrahic) [inline docs/experience/resources.md]: Hits is => **Hits** -

Comment (adnanrahic) [reply docs/experience/resources.md]: Same comment for all the bullets here. Change "N is" to "**N** -"

File: docs/experience/resources.md [hunk 14]
```diff
@@ -0,0 +1,70 @@
...
+ * Total is the average total time it took to fetch the resource. This time also includes the time the resource spent in the browser queue for fetching.
+
+<img
+  class="content-modal-image"
+  alt="Resources Overview Table Screenshot"
+  src="../../images/experience/resources.png"
+  title="Resources Overview"
+/>
+
+Some of these timings are not available for CORS resources (see below).
```
Comment (adnanrahic) [inline docs/experience/resources.md]: Some of these timings are not available for CORS resources (see below).
=>
Some of these timings are not available for CORS resources as you can see below.

**Comment by amir-hadzic** [comment]
I've made the adjustments:

localhost_8000_experience_resources_ (1)

File: docs/experience/resources.md [hunk 15]
```diff
@@ -0,0 +1,70 @@
+title: Resource Timing
+description: How Sematext Experience collects external resource data such as downloaded images, CSS files, and JavaScript files
+-----
+
+Experience collects resource timing data such as downloaded images, CSS and JavaScript files. When someone visits your website the browser will begin downloading resources. These resources can have a big impact on your website's loading speed.
+
+The same process happens in single-page apps when user navigates to a different part of the web app.
```
Comment (adnanrahic) [inline docs/experience/resources.md]: The same process happens in single-page apps when user navigates to a different part of the web app.
=>
The same process happens in single-page apps when users navigate to a different part of the web app.

File: docs/experience/resources.md (hunk 7 above)
Comment (adnanrahic) [reply docs/experience/resources.md]: Did you miss this one? :smile:

**Pull request merged**